dqgen/resources/query_templates/movement_cross_instance.rq

```diff
@@ -25,7 +25,9 @@ PREFIX xhv: <http://www.w3.org/1999/xhtml/vocab#>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 # Show all movements cross instances
-SELECT DISTINCT ?oldInstance ?oldInstancePrefLabel ?oldInstancePrefLabelLang ?newInstance ?newInstancePrefLabel ?newInstancePrefLabelLang ?property ?value ?valueLang
+SELECT DISTINCT ?oldInstance (?class as ?oldInstanceClass) ?oldInstancePrefLabel ?oldInstancePrefLabelLang ?newInstance
```

Comment:
I would say we do not need ?oldInstanceClass and ?newInstanceClass, ?class is enough